policyengine_it/variables/household/person/is_dependent_between_18_and_21.py

```diff
@@ -0,0 +1,15 @@
+from policyengine_it.model_api import *
+
+
+class is_dependent_between_18_and_21(Variable):
```

Comment:
We don't typically create variables with values hard coded. Are 18 and 21 policy parameters for some purpose?

Reply:
They factor into Italian social programs - dependents aged 18 to 21 (and also 21 to 24) qualify for different benefits

Reply:
Meeting with Pavel now, who's helping me to better understand how to structure these